src/DependencyInjection/Configuration.php

```diff
@@ -167,4 +169,22 @@ private function addMercureConfiguration(ArrayNodeDefinition $node): void
                 ->end()
             ->end();
     }
+
+    private function addAssetDownloadLimits(ArrayNodeDefinition $node): void
+    {
+        $node->children()
```

Comment:
Can you maybe add a short documentation about this configuration, with an example?